### ISSUE TITLE
DevTools widget tree is not keyboard accessible

### ISSUE THREAD

**Issue by nbros** [flutter/devtools#931]
The arrow keys cannot be used to select a widget in the widget tree in the Dart DevTools page : the arrow keys scroll the tree view instead of changing the selection to the next or previous widget.
(in Chrome 76)
![image](https://user-images.githubusercontent.com/141089/63282936-37efb600-c2b0-11e9-9d57-b2fe5ad03433.png)

**Comment by adil-s** [comment]
the widget tree isn't screen reader friendly as I can't comprehend the hierarchy of the widget tree. I can't figure out what widget is parent and what widgets are children.  DevTools uses some sort of arrows to define the relationship between parent and child which is completely inaccessible for screen reader user. 
I'm sure there are other issues as well, but I would like to have this issue fix first.
